kafka/client_async.py

```diff
@@ -226,7 +232,7 @@ def __init__(self, **configs):
 
         self._selector.register(self._wake_r, selectors.EVENT_READ)
         self._idle_expiry_manager = IdleConnectionManager(self.config['connections_max_idle_ms'])
-        self._closed = False
+
```

Comment:
Wouldn't you want to delete these new lines?

Reply:
👍